internal/pkg/cli/svc_deploy.go

```diff
@@ -425,6 +426,12 @@ func (o *deploySvcOpts) deploySvc(addonsURL string) error {
 
 	if err := o.svcCFN.DeployService(conf, awscloudformation.WithRoleARN(o.targetEnvironment.ExecutionRoleARN)); err != nil {
 		o.spinner.Stop(log.Serrorf("Failed to deploy service.\n"))
+		errors, describeErr := o.svcCFN.GetStackErrors(conf)
+		if describeErr != nil {
+			return fmt.Errorf("describe stack: %w", describeErr)
+		}
+		log.Infoln("Cloudformation status reason:")
+		log.Infof("%s\n", errors[0].StatusReason)
```

Comment:
What do you think of this?
1. If the call to `o.svcCFN.GetStackErrors(conf)` fails, let's also return the original reason as well as the follow-up error:
   ```go
   return fmt.Errorf("deploy service: %v: describe error events %v", err, eventsErr)
   ```
2. Instead of logging status reason can we suffix it to the error message?
   ```go
   return fmt.Errorf("deploy service: %v: %v", err, events[0].StatusReason) 
   ```

Reply:
I wonder if we can add this error handling logic to `DeployService` error handling since it's the same for both "job deploy" and "svc deploy"